web/src/components/common/userComponents/ProfileEditor.jsx

```diff
@@ -173,14 +173,29 @@ export default class ProfileEditor extends Component {
             </div>
           </div>
           <div className="row">
-            <label className="col-2">New Password</label>
+            <label className="col-2">Old password</label>
+            <div className="col-4">
+              <TextField
+                          value={this.state.pw_old}
+                          type="password"
+                          name="pw_old"
+                          hintText="Your old password"
+                          onChange={this.handleInputChange}
+                          errorText ={this.state.pw_new.length === 0 ? "Password can not be empty" :""} />
```

Comment:
yes, but this is the old pw field and you are checking if `this.state.pw_new.length` is not 0